service/tests/prelude.repl

```diff
@@ -1,5 +1,7 @@
 //import CanCan = "ai7t5-aibaq-aaaaa-aaaaa-c";
 import CanCan = "rrkah-fqaaa-aaaaa-aaaaq-cai";
+// assume the same identity as a `dfx call` command
+identity default "~/.config/dfx/identity/default/identity.pem";
```

Comment:
@chenyan-dfinity CanCan privileges the original installer of the canister as its admin.  So, to fix the tests, I'm trying to use the same identity as that of the dfx CLI, and thus, the CanCan admin.  That should overcome the access control issues that result in all calls giving back `null`.

But, I'm doing something wrong here still?

Reply:
This looks good to me, the repl is loading the identity.

The error is on AbuseFlag test, maybe the test result is not updated there?

Reply:
hmm, the log shows that in each test file, it's loading a different identity...not sure why

Reply:
Not sure where `~` points to in CI. Maybe try `/home/runner/.config/dfx/identity/default/identity.pem`

Reply:
Okay, will try now...

Here's the ID that I mentioned, in the context of the log output on CI:
  https://github.com/dfinity/cancan/runs/2633337608#step:5:179

Reply:
> You are installing via a wallet. The controller is the wallet. 

I guess so?

It seems like this is something that dfx just does now automatically as part of the build process, along the way to creating the canister ID and building it.  Or perhaps our dfx.json file is set to do this.  I've lost track of how the local build works in detail, or why it needs to have a wallet canister, etc.

Reply:
> Let me think about how to express this in ic-repl.

Okay, sounds good.

Reply:
Yes. There are two options: 1) Only run `dfx build` and let `ic-repl` installs the wasm. This can be useful for testing canister upgrade in the future; 2) Proxy all the calls in `ic-repl` via wallet. I will need another ic-repl release for this.

Reply:
I disabled tests that don't pass. @matthewhammer is working on fixing them in https://github.com/dfinity/cancan/pull/22